src/main/resources/listLookup.json

```diff
@@ -557,5 +557,96 @@
         "restrictedProvenances": [],
         "defaultSensitivity": "",
         "isNonStrategic": true
+    },
+    "SSCS_MIDLANDS_DAILY_HEARING_LIST": {
+        "friendlyName": "Midlands First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+        "welshFriendlyName": "Rhestr o Wrandawiadau Dyddiol Tribiwnlys Haen Gyntaf Canolbarth Lloegr (Nawdd Cymdeithasol a Chynnal Plant)",
+        "shortenedFriendlyName": "SSCS Midlands Daily Hearing List",
+        "url": "sscs-midlands-daily-hearing-list",
+        "jurisdictions": ["Tribunal"],
+        "welshJurisdictions": ["Tribiwnlys"],
+        "restrictedProvenances": [],
+        "defaultSensitivity": "",
+        "parentPage": "sscs-daily-hearing-list",
+        "isNonStrategic": true,
+        "isWeeklyList": true
+    },
+    "SSCS_SOUTH_EAST_DAILY_HEARING_LIST": {
+        "friendlyName": "South East First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+        "welshFriendlyName": "Rhestr o Wrandawiadau Dyddiol Tribiwnlys Haen Gyntaf De Ddwyrain Lloegr (Nawdd Cymdeithasol a Chynnal Plant)",
+        "shortenedFriendlyName": "SSCS South East Daily Hearing List",
+        "url": "sscs-south-east-daily-hearing-list",
+        "jurisdictions": ["Tribunal"],
+        "welshJurisdictions": ["Tribiwnlys"],
+        "restrictedProvenances": [],
+        "defaultSensitivity": "",
+        "parentPage": "sscs-daily-hearing-list",
+        "isNonStrategic": true,
+        "isWeeklyList": true
+    },
+    "SSCS_WALES_AND_SOUTH_EAST_DAILY_HEARING_LIST": {
+        "friendlyName": "Wales and South West First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+        "welshFriendlyName": "Rhestr o Wrandawiadau Dyddiol Tribiwnlys Haen Gyntaf Cymru a De Orllewin Lloegr (Nawdd Cymdeithasol a Chynnal Plant)",
+        "shortenedFriendlyName": "SSCS Wales and South East Daily Hearing List",
```

Comment:
The friendly name says South West and here is South East

Reply:
changed.